src/osx/carbon/font.cpp

```diff
@@ -819,6 +819,19 @@ void wxNativeFontInfo::InitFromFontDescriptor(CTFontDescriptorRef desc)
 
     wxCFTypeRef(CTFontDescriptorCopyAttribute(m_descriptor, kCTFontFamilyNameAttribute)).GetValue(m_familyName);
     wxCFTypeRef(CTFontDescriptorCopyAttribute(m_descriptor, kCTFontNameAttribute)).GetValue(m_postScriptName);
+    AdjustPostScriptName();
+}
+
+void wxNativeFontInfo::AdjustPostScriptName()
+{
+    if ( wxCheckOsVersion( 10, 16) )
```

Comment:
Interesting that [Trac comment](https://trac.wxwidgets.org/ticket/18878#comment:9) claims that the same problem also happens with 10.15. Could we also have `.SFNS`-prefixed fonts there? Or is it due to something completely different on that system?

Reply:
these names have been around for long times, but they didn't cause problems before. I wrote in that ticket that it must be a slightly different situation which I'd like to understand first before blindly applying this to 10.15 as well ..